evalml/model_understanding/prediction_explanations/_user_interface.py

```diff
@@ -687,9 +724,12 @@ def make_dataframe(self, index, y_pred, y_true, scores, dataframe_index):
 
 
 class _SHAPTable(_SectionMaker):
-    def __init__(self, top_k_features, include_shap_values):
+    def __init__(
+        self, top_k_features, include_shap_values, include_expected_value=False
+    ):
         self.top_k_features = top_k_features
         self.include_shap_values = include_shap_values
+        self.include_expected_value = include_expected_value
```

Comment:
I don't think the value of this parameter is actually used in the implementation. I vote we remove it from the function signature.